include/mbedtls/aes.h

```diff
@@ -77,7 +77,9 @@ typedef struct mbedtls_aes_context {
     size_t MBEDTLS_PRIVATE(rk_offset);           /*!< The offset in array elements to AES
                                                     round keys in the buffer. */
 #if defined(MBEDTLS_AES_ONLY_128_BIT_KEY_LENGTH) && !defined(MBEDTLS_PADLOCK_C)
-    uint32_t MBEDTLS_PRIVATE(buf)[44];
+    uint32_t MBEDTLS_PRIVATE(buf)[44];           /*!< Aligned data buffer to hold
+                                                    round keys in the 128-bit case for
+                                                    10 rounds calculation. */
```

Comment:
nit: Maybe "10 rounds of calculation" sounds a bit better. Otherwise though, thanks for this change.

Reply:
Err, it's actually a calculation of 10 round keys.

We probably just need to say "Aligned data buffer to hold 10 round keys for 128-bit case"